provision-contest/ansible/judgehost.yml

```diff
@@ -58,7 +58,7 @@
 
     - name: Disable all cores
       service:
-        name: "domjudge-judgedaemon@{{ item }}"
```

Comment:
why does this change?

Reply:
Because Jaap found a mistake in what I copied so it makes sense to fix it in both locations.